neard/src/runtime/mod.rs

```diff
@@ -315,6 +314,10 @@ impl NightshadeRuntime {
         }
     }
 
+    pub fn migration_data(&self) -> Arc<MigrationData> {
```

Comment:
where is this used?

Reply:
I use it in `protocol_feature_restore_receipts_after_fix_tests` test in `process_blocks`, to extract migration data from runtime directly and avoid unnecessary data loading.

Reply:
If it is used in test please either decorate it with #[cfg(test)] or find other ways to implement it so that it does not get used accidentally anywhere else.

Reply:
It's problematic to use `#[cfg(test)]` currently: https://github.com/near/nearcore/issues/4305
So I used `load_migration_data`.